packages/nvidia_nat_ragas/src/nat/plugins/ragas/rag_evaluator/register.py

```
@@ -93,22 +96,20 @@ def metric_config(self) -> RagasMetricConfig:
+        """Fetch metric constructor from the v0.4 collections namespace."""
+        module_names = ("ragas.metrics.collections", )
+        for module_name in module_names:
+            try:
+                module = import_module(module_name)
+            except ImportError:
+                continue
+            metric_ctor = getattr(module, metric_name, None)
+            if metric_ctor is not None:
+                return metric_ctor
+        raise ValueError(
+            f"Ragas metric '{metric_name}' was not found in supported namespaces: {', '.join(module_names)}")
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)